physcraper/__init__.py

```diff
@@ -1834,26 +1869,30 @@ def read_local_blast_query(self, fn_path):
                 bitscore = float(bitscore)
                 stitle = salltitles
                 # NOTE: sometimes there are seq which are identical & are combined in the local blast db...
-                # Get all of them! (get redundant seq info)
+                # Get all of them! (they can be of a different taxon ids = get redundant seq info)
                 found_taxids = set()
                 found_spn = set()
-                if len(sallseqid.split(";")) > 1:
-                    # get additional info only for seq that pass the eval
-                    if evalue < float(self.config.e_value_thresh):
+                # get additional info only for seq that pass the eval
+                if evalue < float(self.config.e_value_thresh):
+                    if len(sallseqid.split(";")) > 1:
```

Comment:
Pulling out this search for merged tax_ids into as separate function would make this section easier to read and debug. If I'm understanding correctly, this if statement is only true if there is more that one id for the sequence? It is hard to find where the 'else' to this 'if' is, which is a sign that this whole block should be pulled out into a seperate function.